sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -228,4 +234,126 @@ public HttpPipeline getHttpPipeline() {
             });
     }
 
+
+    //==================================================================================================================================================
```

Comment:
Is supposed to be the java equivalent of `#region`? 😉

Reply:
I just wanted to have some API separation to ease locating different APIs, we will remove them once we are ready to ship, I don't feel strongly about this at all though, so if you guys hate it we can totally remove it.